test/extensions/filters/network/redis_proxy/mocks.cc

```diff
@@ -8,34 +8,11 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace RedisProxy {
 
-MockRouter::MockRouter() = default;
```

Comment:
Sorry can you revert all of these changes, gmock recommends this pattern to improve compile performance (see their docs), so we do this everywhere.

Reply:
I'll revert the changes here, and copy the same pattern for the new mocks.cc and mocks.h in test/extensions/common/redis.